modules/identity-provider-about-request-header.adoc

```diff
@@ -20,13 +25,13 @@ requests for OAuth tokens to the proxy endpoint that proxies to
 
 To redirect unauthenticated requests from clients expecting browser-based login flows:
```

Comment:
I think this might be better as a procedure module.

Reply:
Yeah I agree the module is formatted oddly. Though since this update is for a BZ that was escalated by a customer, I'm not going to worry about it for now. I'll make a note to take a look at it later if I have time. Thanks!